### DISCUSSION TITLE
Album Tagging

### DISCUSSION THREAD

**Discussion by floydp386** [MoleMan1024/audiowagon#109] [General]
I use MP3Tag for all my music files
Today I noticed that some of my my albums came up under the "Various Artists" folder on AudioWagon
Even though they have been tagged correctly with Album name and Album Artist 
See picture (Hit Parade, Lungs, Trench etc)
I even removed them from my usb stick, deleted the database, reindexed and added them again after checking they were tagged correctly and it did the same thing - strange???

![20230222_115531_resized](https://user-images.githubusercontent.com/116339566/220663325-a03b3c6f-417f-4621-acb6-9a8665901a91.jpg)

The only music folders that should be here are Furious 7 and UK Top 100 singles as they both have "Various Artists" in the Album artist field

**Comment by MoleMan1024** [comment]
Did you check if those albums have the "compilation" flag set? See [the Frequently Asked Questions](https://moleman1024.github.io/audiowagon/faq.html#why-do-my-compilation-albums-show-up-as-separate-albums)

**Reply by floydp386** [reply]
![Screenshot 2023-02-22 171405](https://user-images.githubusercontent.com/116339566/220704864-728c69c1-8d3c-4a81-8dbb-0509b1ae2474.jpg)

Spot on - Just spotted it. 
I will remove this and all should be sorted

**Comment by floydp386** [comment]
![20230222_191415_resized](https://user-images.githubusercontent.com/116339566/220737263-8120464a-cd50-45f6-b2fc-32154c1a243c.jpg)

One final question.
I can't figure out why I'm seeing 2 Various Artists folders, both have 2 compilation albums in them even though one says 0, 0
I can see one is uppercase A and one is lower case a.
But on MP3Tag its only uppercase A



![Screenshot 2023-02-22 192734](https://user-images.githubusercontent.com/116339566/220737880-72f05924-cf6c-484f-aee7-c3fee84f869d.jpg)

**Reply by MoleMan1024** [reply]
I think there is a small bug in there, it should not show 0 when there are albums inside. I will investigate in https://github.com/MoleMan1024/audiowagon/issues/110

**Reply by MoleMan1024** [reply]
Also from typography point of view, this is not pretty XD

**Reply by floydp386** [reply]
Great, thanks for looking at this.
Would be great if you can sort it
I have a bit of OCD when it comes to my music being sorted properly

**Reply by floydp386** [reply]
Will the app on the AAOS tell me when there is an update?
Is the current version 2.3.5?

**Reply by MoleMan1024** [reply]
Latest version is 2.3.6, see [changelog](https://github.com/MoleMan1024/audiowagon/blob/master/CHANGELOG.md). It should automatically update via Google Play Store in the car (unless you have turned off that feature, then you will need to update manually).

**Reply by floydp386** [reply]
I checked and I am on version 2.3.6
Not sure how to check if I have selected auto updates via Play store

**Reply by MoleMan1024** [reply]
If you are not sure how to check for auto updates it is unlikely you have ever turned it off. Next version of the app will improve on "various artists".
